app/models/commit.rb

```diff
@@ -12,6 +12,9 @@ class Commit < ActiveRecord::Base
   scope :unchecked, -> { where(state: nil) }
   scope :verified, -> { where(state: "verified") }
   scope :unverified, -> { where("commits.state != 'verified' or commits.state is null") }
+  scope :deployed, -> { where(deploy_status: "succeeded") }
+  scope :deployed_but_not_verified, -> { deployed.where("commits.state != 'failed'").unverified }
```

Comment:
Can also do this based on the found "state" values -

```suggestion
  scope :deployed_but_not_verified, -> { deployed.unchecked }
```